### ISSUE TITLE
Add setting to control inline chat accessibility logic

### ISSUE THREAD

**Issue by jrieken** [microsoft/vscode#205714]
We have added an accessible diff viewer for inline chat. Today, this is on when accessibility mode is on and off otherwise. We should have dedicated setting with on, off, and auto

**Comment by jrieken** [comment]
To verify

* check that `inlineChat.accessibleDiffView` exists
* check that it works as advertised
